src/main/java/com/snowflake/kafka/connector/records/RecordService.java

```diff
@@ -108,13 +109,12 @@ public RecordService(SnowflakeTelemetryService telemetryService) {
   /** Record service with null telemetry Service, only use it for testing. */
   @VisibleForTesting
   public RecordService() {
-    this.telemetryService = null;
+    this(null);
```

Comment:
set default here?

Reply:
I don't think that is necessary as it its set to default on line 75?